pkg/node/requester.go

```diff
@@ -212,24 +212,21 @@ func NewRequesterNode(
 		translationProvider = translation.NewStandardTranslatorsProvider()
 	}
 
+	installationID := system.InstallationID()
+	var instanceID string
+	if sysmeta, err := fsr.SystemMetadata(); err == nil {
+		instanceID = sysmeta.InstanceID
+	}
```

Comment:
we should still log (trace or debug)) in the event this fails, for our own sake when/if debugging.